server/routerlicious/packages/lambdas/src/alfred/index.ts

```diff
@@ -801,7 +840,7 @@ export function configureWebSocketServices(
 						clientId,
 					};
 					const throttleError = checkThrottleAndUsage(
-						submitSignalThrottler,
+						throttlersMap?.get("perCluster")?.get("submitSignal"),
 						getSubmitSignalThrottleId(clientId, room.tenantId),
```

Comment:
I think this should be "submitSignals". perCluster throttleId should not be based on clientId+tenantId right?

Reply:
Got it. This should be the tenant level. :)

Reply:
Changed to tenant level.